Public/Src/Cache/ContentStore/Library/Logging/FileLog.cs

```diff
@@ -179,7 +179,7 @@ public virtual void Write(DateTime dateTime, int threadId, Severity severity, st
                     message
                 );
 
-            WriteLineInternal(severity, line);
+            WriteLine(severity, severity.ToString(), line);

```

Comment:
Maybe leave a comment saying that it's important to call `WriteLine` and not `WriteLineInternal` because subclasses may override `WriteLine` to add some extra processing.